.idea/dictionaries/devagrawal.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="devagrawal">
+    <words>
+      <w>cech</w>
+      <w>tangeman</w>
```

Comment:
Android studio kept bugging me, saying that these words are "typos", so I added them to the dictionary. Sorry I'm a slight OCD, swiggly lines freak me out.

Reply:
Hmmmm, a neat trick.